scripts/build/build-e2fsprogs-guest.sh

```
@@ -0,0 +1,179 @@
+        cflags='-O0 -g3 -fno-omit-frame-pointer -fno-pie'
+        ldflags='-no-pie'
+    fi
+
+    host="${target/-unknown/}"
+    jobs=$(getconf _NPROCESSORS_ONLN 2>/dev/null || echo 4)
+    work=$(mktemp -d "$work_parent/.guest-tools-work.XXXXXX")
+    stage=$(mktemp -d "$output_parent/.guest-tools-stage.XXXXXX")
+    chmod 0755 "$stage"
+    mkdir -p "$work/build"
+
+    local -a configure_args=("--build=$($source/config/config.guess)" "--host=$host" --prefix=/usr
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)